doc/source/whatsnew/v1.1.0.rst

```diff
@@ -13,6 +13,32 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_110.astype_string:
+
+All dtypes can now be converted to ``StringDtype``
```

Comment:
extension dtypes

Reply:
No, all dtypes (this not only about extension dtypes)

Reply:
I'll leave it as it it,